package.json

```
@@ -29,7 +29,7 @@
   },
   "scripts": {
     "generate-readme": "cross-env NODE_ENV=production babel-node ./tools/generate-readme",
-    "build": "webpack --mode production && babel ./src/js/ --ignore '__tests__' --out-dir ./dist --copy-files && cross-env BABEL_ENV=es6 babel ./src/js/ --ignore '__tests__' --out-dir ./dist/es6 --copy-files",
+    "build": "webpack --mode production && babel ./src/js/ --out-dir ./dist --copy-files  && cross-env BABEL_ENV=es6 babel ./src/js/ --out-dir ./dist/es6 --copy-files && rm -r ./dist/utils/portal.js ./dist/es6/utils/portal.js ./dist/__tests__/ ./dist/**/__tests__/ ./dist/**/**/__tests__", 
```

```suggestion
    "build": "webpack --mode production && babel ./src/js/ --out-dir ./dist --copy-files && cross-env BABEL_ENV=es6 babel ./src/js/ --out-dir ./dist/es6 --copy-files && rm -r ./dist/utils/portal.js ./dist/es6/utils/portal.js ./dist/__tests__/ ./dist/**/__tests__/ ./dist/**/**/__tests__",
```

Removing redundant spacing.